dapps/evancore.vue.libs/src/components/dapp-wrapper/dapp-wrapper.ts

```diff
@@ -119,6 +117,15 @@ export default class DAppWrapperComponent extends mixins(EvanComponent) {
    */
   @Prop({ default: true }) createRuntime: boolean;
 
+  get isLoggedin() {
+    console.log('Logged In?', this.$store.state.isLoggedin);
```

Comment:
please remove console logs. If you want `debug` logs, please use blockchain-core logging functions.
```suggestion
```